MapboxCoreNavigation/RouteController.swift

```diff
@@ -351,10 +351,19 @@ extension RouteController: CLLocationManagerDelegate {
         }
 
         // Only check ever 2 minutes for faster route
-        guard location.timestamp.timeIntervalSince(lastLocationDate) >= 120 else { return }
+        guard location.timestamp.timeIntervalSince(lastLocationDate) >= 300 else { return }
         let durationRemaining = routeProgress.durationRemaining
         let currentAlertLevel = routeProgress.currentLegProgress.alertUserLevel
         
+        func compareOldRoute(to route: Route) {
```

Comment:
A “compare” function returns whether something is greater, equal, or less than another thing, whereas this method potentially reroutes the user. A more accurate name would be `rerouteIfFaster(onto:)`.